composer.json

```diff
@@ -8,7 +8,7 @@
   "minimum-stability": "stable",
   "require": {
       "behat/mink": "^1.7.0",
-      "behat/mink-selenium2-driver": "^1.3.1",
+      "behat/mink-selenium2-driver": "^1.3",
```

Comment:
Hey @boznik,
What version should this be?

Reply:
@boznik 
We need to keep this at 1.3.
Otherwise, this fails:

```
  @api @dev @destructive @javascript
  Scenario: Create Featured Block                                                                                                                                                                                       # features/engineering_stanford_bean_types/stanford_bean_types_featured.feature:17
    Given I am logged in as a user with the "administrator" role                                                                                                                                                        # SWSDrupalContext::assertAuthenticatedByRole()
      Form field with id|name|label|value|placeholder "Username" not found. (Behat\Mink\Exception\ElementNotFoundException)
    And I am on "block/add/stanford-featured-block"
```